example/src/App.tsx

```diff
@@ -86,6 +87,7 @@ function App() {
             <TestPageButtons />
             <TestEventButtons />
             <MetamaskButtons />
+            <EthereumEventsButtons />
```

Comment:
Why not just leave it in `MetamaskButtons`?

Reply:
I just thought that it is a little bit different context, so I've decided to split it in order not to grow MetamaskButtons component

Reply:
sounds good